xml/System/Span`1+Enumerator.xml

```diff
@@ -56,9 +55,23 @@ The enumerator does not have exclusive access to the collection; therefore, enum
         <ReturnType>T@</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the item at the current position of the enumerator.</summary>
+        <value>The element in the <see cref="T:System.Span`1" /> at the current position of the enumerator.</value>
+        <remarks>
+      <format type="text/markdown"><![CDATA[  
+
+`Current` is undefined under any of the following conditions: 
```

Comment:
Given that the exception is documented, it's better to say:
`Current` throws an \<xref:System.IndexOutOfRangeException\> under any of the following conditions:

Interesting that's the `IndexOutOfRangeException` rather than `InvalidOperationException`.

Reply:
Yes, it is interesting. And you're right, @pkulikov, about replacing the "undefined" text.